test.js

```diff
@@ -259,6 +259,29 @@ test('`clickElement` option', async t => {
 	await server.close();
 });
 
+test('`inset` option', async t => {
+	const server = await createTestServer();
+
+	server.get('/', async (request, response) => {
+		response.end(`
+			<body style="margin: 0; padding: 40px">
+				<div style="background-color: black; width: 100px; height: 100px;"></div>
+			</body>
+		`);
+	});
+
+	const size = imageSize(await instance(server.url, {
+		scaleFactor: 1,
+		element: 'div',
+		inset: -10
```

Comment:
There should be a separate test for the object-form too, where it uses both negative and positive values, to test that that actually works.

Reply:
another test for object added.